styles.css

```diff
@@ -0,0 +1,259 @@
+@import url("https://fonts.googleapis.com/css2?family=Epilogue:wght@400;700&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: var(--primary-font);
+}
+
+:root {
+  --primary-text: #000000;
+  --secondary-text: #747171;
+  --primary-color: #f9a826;
+  --secondary-color: #ffffff;
+  --btn-color: #965c49;
+  --box-shadow: #979494;
+  --primary-font: "Epilogue", sans-serif;
+  --primary-alert: #80b918;
+  --success-alert: #dee2e6;
+  --info-alert: #4cc9f0;
+  --warning-alert: #d62828;
+}
+
+/* Navigation */
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  box-shadow: 0 0 1rem 0;
+  width: 100%;
+}
+
+.navigation .nav-icon {
+  font-size: 2.5rem;
+  color: var(--primary-color);
+  padding: 0 0 2rem 0;
+  z-index: 2;
+}
+
+.navigation .nav-brand {
+  font-size: 3rem;
+  padding: 0 0 2rem 0;
+  font-weight: bold;
+}
+
+.navigation {
+  padding: 1rem;
+  border-bottom-left-radius: 1rem;
+}
+
+.navigation .nav-pills {
+  text-align: right;
+  margin-left: auto;
+}
+
+.navigation .link:hover {
+  color: var(--primary-color);
+  cursor: pointer;
+}
+
+/* links */
+.link {
+  text-decoration: none;
+  padding: 0.5rem 1rem;
+  color: black;
+}
+
+.list-non-bullet {
+  list-style: none;
+  font-size: 1.5rem;
+  padding-top: 1.5rem;
+}
+
+.list-item-inline {
+  display: inline;
+  padding: 0rem 0.5rem;
+}
+
+/* header */
+
+.hero-header {
+  position: relative;
+  margin-top: 10rem;
+  display: flex;
+  font-family: var(--primary-font);
+}
+
+.hero-text {
+  margin: 3rem 5rem;
+  flex: 1;
+  font-family: var(--primary-font);
+}
+
+.hero-heading {
+  font-size: 3.5rem;
+}
+
+.hero-para {
+  font-size: 1.5rem;
+  font-weight: 300;
+  padding-top: 1rem;
+  line-height: 3rem;
+  color: var(--secondary-text);
+}
+
+.hero-btns {
+  margin-top: 2rem;
+}
+
+.hero-img {
+  width: 50%;
+}
+
+/* buttons */
+.hero-btns {
+  display: flex;
+  flex-direction: row;
+  gap: 2rem;
+}
+
+.btn {
+  font-size: 1.5rem;
+  color: white;
+  padding: 1.5rem;
+  text-decoration: none;
+  border: 2px var(--btn-color);
+  border-radius: 3%;
+  cursor: pointer;
+}
+
+.btn-primary {
+  padding-right: 3rem;
+}
+.hero-btns .btn-primary,
+.btn-secondary {
+  background-color: var(--primary-color);
+}
+
+/* Installation */
+
+.install-heading {
+  font-size: 3.5rem;
+}
+.install-text {
+  font-size: 1.5rem;
+  font-weight: 300;
+  padding-top: 1rem;
+  line-height: 3rem;
+  color: var(--secondary-text);
+}
+
+/* Sidebar */
+
+.main-content {
+  margin: 50px auto;
```

Comment:
you can use relative units if a value is more than 10px.

Reply:
done